packages/components/tooltip/tooltip.stories.js

```diff
@@ -58,8 +58,8 @@ basic.args = {
   offsetLeft: 0,
 };
 
-const BodyStyle_2 = { marginTop: -22, marginLeft: 50, position: "absolute" };
-const BodyStyle_3 = { marginTop: 70, marginLeft: 250 };
+const BodyStyle_2 = {};
```

Comment:
Are you sure that in this case you need to leave an empty object, and not to delete it ?

Reply:
deleted unnecessary constants